### ISSUE TITLE
Noto Symbols: Variations sequences for symbols are not supported

### ISSUE THREAD

**Issue by GoogleCodeExporter** [googlei18n/noto-fonts#321]
```
The following characters have variation sequences defined for them, but they 
are not supported in the Noto Sans Symbols font:

2229 FE00; with serifs; # INTERSECTION 
222A FE00; with serifs; # UNION
2268 FE00; with vertical stroke; # LESS-THAN BUT NOT EQUAL TO
2269 FE00; with vertical stroke; # GREATER-THAN BUT NOT EQUAL TO
2272 FE00; following the slant of the lower leg; # LESS-THAN OR EQUIVALENT TO
2273 FE00; following the slant of the lower leg; # GREATER-THAN OR EQUIVALENT TO
228A FE00; with stroke through bottom members; # SUBSET OF WITH NOT EQUAL TO
228B FE00; with stroke through bottom members; # SUPERSET OF WITH NOT EQUAL TO
2293 FE00; with serifs; # SQUARE CAP
2294 FE00; with serifs; # SQUARE CUP
2295 FE00; with white rim; # CIRCLED PLUS
2297 FE00; with white rim; # CIRCLED TIMES
229C FE00; with equal sign touching the circle; # CIRCLED EQUALS
22DA FE00; with slanted equal; # LESS-THAN EQUAL TO OR GREATER-THAN
22DB FE00; with slanted equal; # GREATER-THAN EQUAL TO OR LESS-THAN
2A3C FE00; tall variant with narrow foot; # INTERIOR PRODUCT
2A3D FE00; tall variant with narrow foot; # RIGHTHAND INTERIOR PRODUCT
2A9D FE00; with similar following the slant of the upper leg; # SIMILAR OR 
LESS-THAN
2A9E FE00; with similar following the slant of the upper leg; # SIMILAR OR 
GREATER-THAN
2AAC FE00; with slanted equal; # SMALLER THAN OR EQUAL TO
2AAD FE00; with slanted equal; # LARGER THAN OR EQUAL TO
2ACB FE00; with stroke through bottom members; # SUBSET OF ABOVE NOT EQUAL TO
2ACC FE00; with stroke through bottom members; # SUPERSET OF ABOVE NOT EQUAL TO

(This is contrary to Mongolian and Phags-pa, where the sequences are supported 
in the fonts.)

The glyphs for the above sequences (shown at 
http://www.unicode.org/Public/UCD/latest/ucd/StandardizedVariants.html) should 
be added to the Noto Symbols font, and a cmap subtable format 14 should be 
added to map the sequences to glyphs.

(CC-ing Behdad to confirm that cmap subtable format 14 is preferable to a GSUB 
rule.)
```

Original issue reported on code.google.com by `roozbeh@google.com` on 2 Apr 2015 at 12:16

**Comment by behdad** [comment]
cc @roozbehp @kmansourMT @behdad

**Comment by behdad** [comment]
Yes, for everything other than the Mongolian variation selectors, cmap 14 is required.

**Comment by dougfelt** [comment]
PhagsPa also uses GSUB to implement variants.

**Comment by dougfelt** [comment]
This is fixed with NotoSansSymbols v1.07.

On my system pango-view doesn't do the substitution, but hb-view works fine-- I'm not set up properly.  So I used hb-view on some characters and looked at the variation selector cmap.
